pkg/operator/controller/ingress/controller.go

```diff
@@ -699,10 +699,18 @@ func (r *reconciler) ensureIngressController(ci *operatorv1.IngressController, d
 		return fmt.Errorf("failed to ensure cluster role: %v", err)
 	}
 
-	if err := r.ensureRouterNamespace(); err != nil {
+	if _, _, err := r.ensureRouterNamespace(); err != nil {
 		return fmt.Errorf("failed to ensure namespace: %v", err)
 	}
 
+	if err := r.ensureRouterServiceAccount(); err != nil {
+		return fmt.Errorf("failed to ensure service account: %v", err)
+	}
+
+	if err := r.ensureRouterClusterRoleBinding(); err != nil {
+		return fmt.Errorf("failed to ensure cluste role binding: %v", err)
```

Comment:
Missing an "r", and might as well use `%w`:
```suggestion
		return fmt.Errorf("failed to ensure cluster role binding: %w", err)
```

Reply:
fixed. Thanks!